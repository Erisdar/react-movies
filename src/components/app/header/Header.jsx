import BrandTitle from 'app/brandTitle/BrandTitle';
import React from 'react';
import './header.css';

export default () => (
    <div className="header justify-content-center">
        <BrandTitle className="header__brand-title brand-title_color-red" />
    </div>
);