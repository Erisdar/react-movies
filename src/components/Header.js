import React from 'react';
import BrandTitle from './BrandTitle';
import '../styles/header.css';

export default () => (
    <div id="header" className="justify-content-center">
        <BrandTitle />
    </div>
);