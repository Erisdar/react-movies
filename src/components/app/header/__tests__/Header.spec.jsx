import { shallow } from 'enzyme';
import React from 'react';
import Header from '../Header';

describe('Header component', () => {
    it('should be render correctly', () => {
        const component = shallow(<Header />);
        expect(component).toMatchSnapshot();
    });
});