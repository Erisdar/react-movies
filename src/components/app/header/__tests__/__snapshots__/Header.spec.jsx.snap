// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should be render correctly 1`] = `
<div
  className="header justify-content-center"
>
  <_default
    className="header__brand-title brand-title_color-red"
  />
</div>
`;
